libvmi/debug.h

```diff
@@ -51,7 +51,8 @@ typedef enum {
     VMI_DEBUG_WRITE     = (1 << 13),
     VMI_DEBUG_DRIVER    = (1 << 14),
     VMI_DEBUG_PEPARSE   = (1 << 15),
-    VMI_DEBUG_BAREFLANK = (1 << 16),
+    VMI_DEBUG_ELFPARSE  = (1 << 16),
+    VMI_DEBUG_BAREFLANK = (1 << 17),
```

Comment:
Please don't change existing definition numbers, introduce new ones with bits not yet used.

Reply:
Mhh, okay. I would have preferred to keep PEPARSE and ELFPARSE together, especially if it's only a private header. But having ELFPARSE at bit 17 is also fine for me.